data/blog/electron-6-0.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Electron 6.0.0
+author: 
+date: '2019-07-30'
+---
+
+The Electron team is excited to announce the release of Electron 6.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). The release is packed with upgrades, fixes, and new features. We can't wait to see what you build with them! Continue reading for details about this release, and please share any feedback you have!
```

Comment:
> We can't wait to see what you build with them!

This seems like tradition now :green_heart:

Reply:
credits to @codebytere from v3.0.0 ❤️